### PR TITLE
#199 Project Cards showing up, EmptyProjects fixed, Projects refactored and misc fixes

### PR DESCRIPTION
**Pull request by rkzel**
In order to reduce distance to dev branch...

### CODE REVIEW

File: apps/projects/app/components/TabbedView/TabbedView.js [hunk 1]
```diff
@@ -10,19 +10,11 @@ const StyledTabbedView = styled.div`
...
-  selectTabIndex(activeIndex) {
-    this.setState({ activeIndex })
-  }
-
   render() {
     const children = React.Children.map(this.props.children, child => {
       return React.cloneElement(child, {
-        activeIndex: this.state.activeIndex,
-        onSelectTab: this.selectTabIndex.bind(this),
+        activeIndex: this.props.activeIndex,
```
Comment (ottodevs) [inline apps/projects/app/components/TabbedView/TabbedView.js]: Why? Does really the parent component need to have info about the state of this child? In my opinion it is better to encapsulate the behaviour inside the component and don't take it out

**Comment by ottodevs** [comment]
It seems cards need more work before merging this:
image

I am fixing the problems right now

**Comment by ottodevs** [comment]
Here is the result so far:
image

IMHO is ready to be merged

**Comment by Quazia** [comment]
Looks good, I'm going to merge

**Pull request merged**